### ISSUE TITLE
'servers' submodule missing from package in 1.1.1

### ISSUE THREAD

**Issue by Anvil** [pogzyb/asyncwhois#83]
Hey There.

I think we forgot to update the `setup.py` configuration to include the `servers` submodule in the distributed package.

```diff
diff --git a/setup.py b/setup.py
index ffc90ae..aa7f7a1 100644
--- a/setup.py
+++ b/setup.py
@@ -47,7 +47,7 @@ setuptools.setup(
         "Framework :: AsyncIO",
     ],
     url="https://github.com/pogzyb/asyncwhois",
-    packages=["asyncwhois"],
+    packages=setuptools.find_packages(),
     package_dir={"asyncwhois": "asyncwhois"},
     python_requires=">=3.9",
 )
```

`find_packages` should automatically do the trick. Sorry for the misshap.

**Comment by pogzyb** [comment]
Hi - right that was my fault. 
I released 1.1.2 a little while ago with this fix and yanked 1.1.1 from pypi

#81

**Comment by Anvil** [comment]
Ha, alright I'm lagging. Thank you !
